experimental/dds/tree2/src/shared-tree-core/editManager.ts

```diff
@@ -416,12 +416,7 @@ export class EditManager<
 					"Received a sequenced change from the local session despite having no local changes",
 				);
 			this.pushToTrunk(sequenceNumber, { ...newCommit, change }, true);
-
-			{
-				// TODO: Replace the line below with this line when UndoRedoManager is better optimized.
-				// this.localBranch.rebaseOnto(this.trunk, this.trunkUndoRedoManager);
-				this.localBranch.setHead(localPath.reduce(mintCommit, this.trunk.getHead()));
-			}
+			this.localBranch.rebaseOnto(this.trunk);
```

Comment:
@noencke There's something about this change that's causing the ci mocha tests to fail 😢

Reply:
Confirmed that it's nothing in this PR. I'll do a bit more investigation before I just comment it out again.

Reply:
Turns out the error is happening even on your initial change to get this in so maybe it was just being hidden by that timeout you ran into. For now, I'm going to just revert this part and commit without a unit test because this early out will move out of the URM with my next task to factor out the RepairDataStoreProvider anyways.